src/elements/Video/Video.scss

```diff
@@ -43,6 +43,11 @@
                 width: 100%;
             }
         }
+
+        > *:only-child {
+            margin-left: auto;
+            margin-right: auto;
+        }
```

Comment:
Hm... I don't understand how this detects a vertical video? :thinking:

Reply:
I assumed we always want to center the video 👀 since there are no alignment options for that.

Reply:
Okay, but why `*:only-child`? I'm just trying to make sense of it to myself.

Reply:
I played safe here, since iframely injects html and we don't have control over it. So that if at any case it adds another element, we don't accidentally apply this style to it. That's a rare case.

Maybe I should remove it.